challenge_1/python/ukirderohit/reverse.py

```diff
@@ -0,0 +1,2 @@
+s = raw_input("Enter String: ")
+print (s[::-1])
```

Comment:
Assuming Python2 due to the use of raw_input. This should just be `print s[::-1]`.